plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -272,6 +273,7 @@
     private static final int WRITER_VERSION = 2;
     // The highest writer version Trino supports writing to
     private static final int MAX_WRITER_VERSION = 4;
```

Comment:
Why not upgrading `MAX_WRITER_VERSION` to `5` ?

Reply:
because DDL operations not supported.

Reply:
I think we want to bump `MAX_WRITER_VERSION`. Newly created tables will still use `WRITER_VERSION = 2`